diagnostics/ENSO_MSE/MSE_VAR/NCL_general/plot_bars_composite_OBS.ncl

```diff
@@ -0,0 +1,223 @@
+
+      lon1 = getenv("slon1")
+      lon2 = getenv("slon2")
+      lat1 = getenv("slat1")
+      lat2 = getenv("slat2")
+
+       slon1 = tofloat( lon1)
+       slon2 = tofloat( lon2)
+       slat1 = tofloat( lat1)
+       slat2 = tofloat( lat2)
+
+      composite_dir_data  = getenv("ENSO_MSE_WKDIR_COMPOSITE")+"/obs/netCDF/DATA/"
+      varcode  = getenv("POD_HOME")
+       model =  "  " ;   leave  blank for OBS
+
+      parameter_name = varcode + "/shared/parameters.txt"
+      datamode = "Anomaly"
+      data  = asciiread(parameter_name,-1,"string")
+      season = str_get_field(data(10), 3," ")
+
+;;  read  in the data from MSE_VAR
+	
+	modes =  (/ "ELNINO" , "LANINA" /)
+       title_modes = (/ "El Nino", "La Nina" /)
+
+      tlabels = (/ "MSE~B~vert. advect.~N", "M~B~adv~N",  "T~B~adv~N" , "SW~B~net~N", \
+                   "LW~B~net~N", "THF~B~net~N" /)
+
+      do n = 0, 1
+	mode =  modes(n) ;;
+	
+       namein =  getenv("ENSO_MSE_WKDIR_MSE_VAR")+"/obs/netCDF/" +  mode +  "/MSE_general_variance.out"
+;;       nameout = getenv("ENSO_MSE_WKDIR_MSE_VAR")+ "/obs/PS/MSE_VAR_comp_general_" + model 
+      nameout = getenv("ENSO_MSE_WKDIR")+ "/obs/PS/MSE_VAR_comp_general_" + mode
+;;;  construct the main title based on model, season and region
+
+
+       ydim = 8
+       dims = ( [ydim] ) 
+       datain  = fbindirread (namein, 0, dims , "float")
+	datain(0) = factor * datain(0)
+
+        data2       = new( (/  ydim-2 /), float) 
+        data2(0)    =  datain(1)/datain(0)     ;;  omse
+        data2(1)    =  datain(2)/datain(0)     ;;  moist adv. 
+        data2(2)    =  datain(3)/datain(0)     ;;  temp adv.
+        data2(3)    =  datain(4)/datain(0)      ;;   SW
+        data2(4)    =  datain(5)/datain(0)      ;;   Lw  
+        data2(5)    = (datain(6)+datain(7))/datain(0) ;; THF
+
+;======================================================================================	
+;;   wks_type = "png"
+;;   wks_type@wkWidth = 1000
+;;   wks_type@wkHeight = 1000
+   wks_type = "eps"
+   wks = gsn_open_wks( wks_type, nameout)
+   wks_type@wkOrientation = "portrait" ;;  "landscape"
+
+;;   wks = gsn_open_wks ("X11", "scatter")
+   gsn_define_colormap(wks, "grads_default")
+
+;;; controling paramters
+     width=  0.25 ;   width of the bars 
+     nxl =  ydim - 2
+     xl1 = 0.72 + 0.25
+     xl2 = nxl - 1 + xl1
+     ndim1 = 0 
+     ndim2 = ydim -2 -1 ; dimensions to display 
+
+     sres = True
+     sres@tmYLMode   =  "Automatic" ;;   "Manual"
+     sres@vpWidthF = 0.8 ;; 0.9 ;;  0.7
+     sres@vpHeightF = 0.5
+     sres@vpXF = 0.10 ;;  0.15
+     sres@vpYF = 0.8
+
+     sres@trXMinF = 0
+     sres@trXMaxF = ydim - 1
+;;     sres@trYMinF = ymin
+;;     sres@trYMaxF = ymax
+;;    sres@tmYLTickSpacingF  = dyy
+     sres@gsnDraw = True
+     sres@gsnFrame = False
+;;;    manual labels 
+;;      sres@tmYLMode          = "Manual"
+;;      sres@tmYLTickEndF      = sres@trYMaxF
+;;      sres@tmYLTickStartF    = sres@trYMinF
```

Comment:
remove commented-out lines if not needed

Reply:
lines removed